draft-ietf-tls-tls13.md

```diff
@@ -4073,10 +4073,10 @@ a non-zero octet in the cleartext, it MUST terminate the
 connection with an "unexpected_message" alert.
 
 The presence of padding does not change the overall record size limitations
-- the full encoded TLSInnerPlaintext MUST not exceed 2^14 octets. If the maximum
-fragment length is reduced, as for example by the max_fragment_length extension
-from [RFC6066], then the reduced limit applies to the full plaintext,
-including the padding.
+- the full encoded TLSInnerPlaintext MUST not exceed 2^14 + 1 octets. If the
```

Comment:
nit: please capitalize the "not", per RFC 2119

Reply:
done (+ another case)